dist/index.js

```diff
@@ -1,782 +1,781 @@
+module.exports =
 /******/ (() => { // webpackBootstrap
```

Comment:
why *.js filed included

Reply:
This is the compiled file which is called by Github to run the action. All actions have this file